src/Protocol/SmtpPluginManager.php

```diff
@@ -9,22 +9,20 @@
 
 namespace Zend\Mail\Protocol;
 
-use Zend\ServiceManager\AbstractPluginManager;
```

Comment:
@ezimuel We're already implementing it in the abstract plugin manager implementations; essentially, if the second argument is called, we proxy to the `build()` method (which explicitly defines the argument, though still makes it optional). As such: go for it.